src/pages/responsible-trading/_trading-responsibly.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react'
+import styled from 'styled-components'
+import DontBorrow from 'images/svg/dont-borrow.svg'
+import FreeDemo from 'images/svg/free-demo.svg'
+import SetLimitLosses from 'images/svg/set-limit-chart.svg'
+import BadJudgement from 'images/svg/bad-judgement.svg'
+import Info from 'images/svg/info.svg'
+import { SectionContainer, Flex, Show } from 'components/containers'
+import { Header, Text, Divider, LocalizedLinkText } from 'components/elements'
+import { localize, Localize } from 'components/localization'
+import device from 'themes/device'
+
+const Card = styled.section`
+    width: ${(props) => (props.width ? props.width : 'auto')};
+    min-width: 38.4rem;
+    height: ${(props) => (props.height ? props.height : 'auto')};
```

Comment:
we can use Flex here, and add CardStyle from card component

import { CardStyle } from 'components/elements' 
const Card = styled(Flex)`
  ${CardStyle}
`